lib/features/thread/presentation/thread_controller.dart

```diff
@@ -100,17 +101,18 @@ class ThreadController extends BaseController {
   final CachingManager _cachingManager;
 
   final listEmailDrag = <PresentationEmail>[].obs;
-
+  bool shiftSelectedMode = false;
   bool canLoadMore = true;
   bool canSearchMore = true;
   bool _isLoadingMore = false;
     get isLoadingMore => _isLoadingMore;
   MailboxId? _currentMailboxId;
   jmap.State? _currentEmailState;
   final ScrollController listEmailController = ScrollController();
+  final FocusNode focusNodeKeyBoard = FocusNode();
+  final latestEmailAction = Rxn<PresentationEmail>();
```

Comment:
not make sense in the name of variable. you mean the latest in the range of selection?

Reply:
@hoangdat No, I mean the last email I interacted with (selected or un-selected).